presubmit_tests/test_pr.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### Transferring shell commands used in https://gist.github.com/jcnars/26b8526cde45a1bb5778a36abe90b96b
+### into this test script
+### context / reference: (internal) at: http://b/202240337#comment22
+
+# set up ssh from pod to BMX host
+# using sydney for initial testing
+mkdir /root/.ssh
```

Comment:
I would suggest:
- Isuing the install command to create/chmod in one command
- Using ~ in case this is ever run as non-root
- Removing argument from ssh-keyscan in case there is no ecdsa host key on the other host

bms_host=172.16.30.1

install -d -m 0700 ~/.ssh
ssh-keyscan "${bms_host}" > ~/.ssh/known_hosts

Reply:
Thanks, done